packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -565,6 +568,23 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         const method: N.TsMethodSignature = nodeAny;
         this.tsFillSignature(tt.colon, method);
         this.tsParseTypeMemberSemicolon();
+        if (method.kind === "get") {
+          if (method.parameters.length > 0) {
+            this.raise(this.state.pos, Errors.BadGetterArity);
+          }
+        } else if (method.kind === "set") {
+          if (method.parameters.length < 1) {
```

Comment:
If there are two parameters we should throw
```suggestion
          if (method.parameters.length !== 1) {
```